tests/test_kem.c

```diff
@@ -20,6 +25,8 @@
 #define OQS_TEST_CT_DECLASSIFY(addr, len)
 #endif
 
+#define MLKEM_SECRET_LEN      32
```

Comment:
Yes, correctly pointed as below, the test is specific to ml-kem.

Reply:
Although the secret length is 32 bytes across all ML-KEM versions, the shared secret macro in the header is tied to specific ML-KEM versions. This can lead to the macro being undefined if those versions are not enabled. Therefore, the macro has been defined independently of the ML-KEM version.

Was also discussed here : https://github.com/open-quantum-safe/liboqs/pull/1947#discussion_r1808667708

Reply:
With the newly added guard enabling this only when ML-KEM in general is enabled (and the recognition that ML-KEM is now standardized, so an upstream change breaking this for a single algorithm is _highly_ unlikely), I'm fine with this as-is.